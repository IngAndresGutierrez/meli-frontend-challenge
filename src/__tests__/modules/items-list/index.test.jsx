import { render } from '@testing-library/react'

import ItemList from '../../../modules/items-list/components/ItemsList'

test('renders ItemList correctly', () => {
  const { asFragment } = render(<ItemList />)
  expect(asFragment()).toMatchSnapshot()
})
