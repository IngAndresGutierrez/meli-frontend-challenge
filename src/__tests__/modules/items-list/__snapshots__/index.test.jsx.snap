// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ItemList correctly 1`] = `
<DocumentFragment>
  <div
    class="items-list"
  >
    <div
      class="MuiGrid-root MuiGrid-container items-list__container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root"
      />
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-8 css-9drkqd-MuiGrid-root"
      />
    </div>
    <div
      aria-hidden="true"
      class="MuiBackdrop-root css-1ji7omv-MuiBackdrop-root"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <span
        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorInherit css-62e83j-MuiCircularProgress-root"
        role="progressbar"
        style="width: 40px; height: 40px;"
      >
        <svg
          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </span>
    </div>
  </div>
</DocumentFragment>
`;
